Source/WebCore/css/CSSValueKeywords.in

```diff
@@ -929,6 +929,7 @@ attachment enable-if=ENABLE_ATTACHMENT_ELEMENT
 borderless-attachment enable-if=ENABLE_ATTACHMENT_ELEMENT
 textarea
 textfield
+-internal-auto-base
```

Comment:
I still have a mild preference for calling this `-internal-appearance`. While that doesn't convey what the arguments correspond to, I think it is a more clear in that at least you know what feature this relates to. (With `light-dark` that problem doesn't exist as the feature is called "dark mode".)